File: CS/CustomService.cs
using DevExpress.Mvvm.UI;
using DevExpress.Xpf.Docking;

namespace DockingThreads
{

    public interface ICustomService {
        void BeginUpdate();
        void EndUpdate();
    }

    public class CustomService : ServiceBaseGeneric<DockLayoutManager>, ICustomService {
        public void BeginUpdate() {
            this.AssociatedObject.BeginUpdate();
        }
        public void EndUpdate() {
            this.AssociatedObject.EndUpdate();
        }
    }
}

File: VB/CustomService.vb
Imports DevExpress.Mvvm.UI
Imports DevExpress.Xpf.Docking

Namespace DockingThreads

	Public Interface ICustomService
		Sub BeginUpdate()
		Sub EndUpdate()
	End Interface

	Public Class CustomService
		Inherits ServiceBaseGeneric(Of DockLayoutManager)
		Implements ICustomService

		Public Sub BeginUpdate() Implements ICustomService.BeginUpdate
			Me.AssociatedObject.BeginUpdate()
		End Sub
		Public Sub EndUpdate() Implements ICustomService.EndUpdate
			Me.AssociatedObject.EndUpdate()
		End Sub
	End Class
End Namespace